python/sdk/test/integration_test.py

```diff
@@ -98,10 +98,7 @@ def test_sklearn(
     with merlin.new_model_version() as v:
         merlin.log_model(model_dir=model_dir)
 
-    resource_request = ResourceRequest(1, 1, "100m", "200Mi")
-    endpoint = merlin.deploy(
-        v, deployment_mode=deployment_mode, resource_request=resource_request
-    )
+    endpoint = merlin.deploy(v, deployment_mode=deployment_mode)
```

Comment:
Thanks for simplifying this! 🙏

Reply:
Yes, correct. deploy() function will use the environment's default_deployment_config: https://github.com/caraml-dev/merlin/pull/471/files#diff-998d1ba4cc54befe776f4bfbc6a88d3fd43a8ea4f22a42754511592947cca06aR83-R84